src/Raven.Client/Documents/Session/Loaders/IIncludeBuilder.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using Raven.Client.Extensions;
+using Raven.Client.Documents.Queries.Facets;
+
+namespace Raven.Client.Documents.Session.Loaders
+{
+    public interface IIncludeOperations<T>
```

Comment:
IIncludeBuilder just in case can return IIncludeOperations, but I'm not sure if we will need it here. Maybe drop this and return IIncludeBuilder inside it?

Reply:
done